src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -141,6 +159,11 @@ public TracerSettings(IConfigurationSource source)
         /// </summary>
         public IntegrationSettingsCollection Integrations { get; }
 
+        /// <summary>
+        /// Gets or sets the global tags, which are applied to all <see cref="Span"/>s.
+        /// </summary>
+        public Dictionary<string, string> GlobalTags { get; set; }
```

Comment:
This should be more robust when accessed from multiple threads. Consider using a [ConcurrentDictionary](https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2).

Reply:
Yep, I agree.